tests/test_datasets.py

```diff
@@ -747,5 +749,79 @@ def test_scanpaths_from_fixations(fixation_indices):
     compare_fixations(sub_fixations, new_sub_fixations, crop_length=True)
 
 
+@pytest.mark.parametrize('attribute_name , attribute_value, invert_match', [('dva',1,False), ('some_strings','n',True), ('some_strings','b',True)])
+def test_filter_stimuli_by_attribute(file_stimuli_with_attributes, fixation_trains, attribute_name, attribute_value, invert_match):
+    new_fixations = pysaliency.Fixations.concatenate((fixation_trains, fixation_trains, fixation_trains[:2]))
+    new_fixations = new_fixations[:]
```

Comment:
why is this needed?

Reply:
I just wanted to make it a bit more complicated, I'll remove that line :)

Reply:
I see :D.